EntranceShuffle.py

```diff
@@ -255,8 +255,8 @@ def build_one_way_targets(world, types_to_include, exclude=(), target_region_nam
                         ('Graveyard Shield Grave -> Graveyard',                             { 'index': 0x035D })),
     ('Grave',           ('Graveyard -> Graveyard Heart Piece Grave',                        { 'index': 0x031C }),
                         ('Graveyard Heart Piece Grave -> Graveyard',                        { 'index': 0x0361 })),
-    ('Grave',           ('Graveyard -> Graveyard Composers Grave',                          { 'index': 0x002D }),
-                        ('Graveyard Composers Grave -> Graveyard',                          { 'index': 0x050B })),
+    ('Grave',           ('Graveyard -> Royal Familys Tomb',                                 { 'index': 0x002D }),
```

Comment:
```suggestion
    ('Grave',           ("Graveyard -> Royal Family's Tomb",                                 { 'index': 0x002D }),
```
repeat in other spots

Reply:
Other regions don't have apostrophes in the name

Reply:
This also needs to match the actual region name.